setup.py

```diff
@@ -55,7 +55,7 @@ def run(self):
 
 setup(
     name='deploy-config-generator',
-    version='2.2.2',
+    version='2.2.3',
```

Comment:
Since this is adding a new feature, this should be a minor version bump instead of a patch-level bump